sc-principles/software-citation-principles.tex

```diff
@@ -85,18 +86,18 @@ \section{Introduction \& motivation}
 It gathered members, and then in June 2015 began studying this, with a number of meetings
 and some off-line work by group members to gather materials documenting existing
 practices in member disciplines; gather materials from workshops and other reports;
-review those materials, identifying overlaps and differences; and draft this resulting document.
-This document will be presented and discussed at the Force2016 Conference \cite{force2016}
+review those materials, identifying overlaps and differences; and draft this resulting document,
+which will be presented and discussed at the Force2016 Conference \cite{force2016}
 in April 2016.  We expect that
-this discussion may lead to a second, final version of this document, and we also
-plan to have a follow-on working group that will work with stakeholders to ensure that
+this discussion may lead to a second, final version, and we also
+plan to have a follow-on working group that will interact with stakeholders to ensure that
```

Comment:
On Wed, Feb 24, 2016 at 07:31:41PM -0800, Daniel S. Katz wrote:
> >  in April 2016.  We expect that
> > -this discussion may lead to a second, final version of this document, and we also
> > -plan to have a follow-on working group that will work with stakeholders to ensure that
> > +this discussion may lead to a second, final version, and we also
> > +plan to have a follow-on working group that will interact with stakeholders to ensure that
> 
> can you change this interact back to work?  otherwise, your changes look fine and I will merge them.

Done.

Frank